fintech-examples/fintech-ui/src/app/dashboard/show-accounts/show-accounts.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { AccountDetails } from '../../api';
+import { Subscription } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { AisService } from '../services/ais.service';
+
+@Component({
+  selector: 'app-show-accounts',
+  templateUrl: './show-accounts.component.html',
+  styleUrls: ['./show-accounts.component.scss']
+})
+export class ShowAccountsComponent implements OnInit, OnDestroy {
+  private accountsSubscription: Subscription;
+  accounts: AccountDetails[];
+  selectedAccount = null;
```

Comment:
it is not necessary to assign a null value here. You could write like:
`selectedAccount: string;`

After variable declaration its value is null but looks cleaner

Reply:
done